station/clients/harbor_client.py

```diff
@@ -10,7 +10,7 @@ class HarborClient:
     def __init__(self, harbor_api_url: str = None, username: str = None, password: str = None):
         # Setup and verify connection parameters either based on arguments or .env vars
 
-        self.url = harbor_api_url if harbor_api_url else os.getenv("HARBOR_URL")
+        self.url = harbor_api_url if harbor_api_url else os.getenv("HARBOR_BASE_URL")
```

Comment:
parse from existing environment variable instead of creating an additional one -> if necessary update readme that variable is required